### PR TITLE
libtrng 4.19 (new formula)

### PR DESCRIPTION
**Pull request by valashko**
Add formula for Tina's Random Number Generator Library -
state of the art C++ pseudo-random number generator library for
sequential and parallel Monte Carlo simulations.

- [+] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [+] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [+] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [+] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libtrng.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  desc "Tina's Random Number Generator Library"
+  homepage "https://www.numbercrunch.de/trng/"
+  url "https://www.numbercrunch.de/trng/trng-4.19.tar.gz"
+  sha256 "36b49961c631ae01d770ff481796c8b280e18c6b6e6b5ca00f2b868533b0492e"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
```
Comment (fxcoudert) [inline Formula/libtrng.rb]: Does `make install` without the prior `make` step work? If so, please do so.

File: Formula/libtrng.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      #include <trng/normal_dist.hpp>
+      int main()
+      {
+        trng::yarn2 R;
+        trng::normal_dist<> normal(6.0, 2.0);
+        (void)normal(R);
+        return 0;
+      }
+    EOS
+    flags = %W[
```
Comment (fxcoudert) [inline Formula/libtrng.rb]: Do not use an array, simply inline the flags:

`system ENV.cxx, "test.cpp", "-o", "test", "-I#{include}", "-L#{lib}", "-ltrng4"`

(and note the added `-L#{lib}`)

**Review by fxcoudert — CHANGES_REQUESTED** [review]
Thanks for the nice contribution! 👏 Just a couple of minor comments, and then this can go in.

Please make the changes and squash into a single commit, with identical commit message (`libtrng 4.19 (new formula)`).

File: Formula/libtrng.rb (hunk 1 above)
Comment (valashko) [inline Formula/libtrng.rb]: Unfortunately, simple `make install` does not work and is not intended to work. Please refer to [installation instructions](https://github.com/rabauke/trng4/blob/master/doc/trng.pdf), page 96.

File: Formula/libtrng.rb (hunk 2 above)
Comment (valashko) [inline Formula/libtrng.rb]: Done.

**Comment by fxcoudert** [comment]
Merged. Thanks again @valashko for your contribution to Homebrew!

**Pull request closed without merging**